src/components/FileUpload/components/FilePreview.tsx

```diff
@@ -47,24 +49,41 @@ const FilePreview: React.FC<FilePreviewProps> = ({
 		}
 	}
 
-	return file.name.endsWith('.pdf') ? (
-		<StyledFilePreview>
-			<Document
-				file={file}
-				onPassword={onPasswordCallback}
-				onLoadSuccess={(): void => {
-					if (passwordQueue.currentItem?.password) {
-						onPassword?.(file.name, passwordQueue.currentItem.password)
-						passwordQueue.remove()
-					}
-				}}
-			>
-				<Page pageNumber={1} width={142} />
-			</Document>
-		</StyledFilePreview>
-	) : (
-		<StyledFilePreview src={file.preview} as="img" />
-	)
+	if (file.name.endsWith('.pdf')) {
+		return (
+			<StyledFilePreview>
+				<Document
+					file={file}
+					onPassword={onPasswordCallback}
+					onLoadSuccess={(): void => {
+						if (passwordQueue.currentItem?.password) {
+							onPassword?.(file.name, passwordQueue.currentItem.password)
+							passwordQueue.remove()
+						}
+					}}
+				>
+					<Page pageNumber={1} width={142} />
+				</Document>
+			</StyledFilePreview>
+		)
+	} else if (file.name.endsWith('.csv')) {
+		return (
+			<StyledFilePreview>
+				<Flex
```

Comment:
The styling is not what I'd expect 😊 Did you test it? Do you need help with that? I would like to style it as I attached in the Linear task. Vertically centered, size of the file size text should be `xs`, the name should not overflow and there should be bigger paddings from left and right.

<img width="626" alt="Screenshot 2023-05-19 at 13 48 33" src="https://github.com/purple-technology/phoenix-components/assets/4357381/65ed5cac-7ca7-476e-a1f7-f78b4a546ae0">

Reply:
Daamn, I made some last minute changes before commit. Deleted some redundant styles (as I thought) and it ended like this 🤦‍♂️ 
Thank you for checking after me, now it should look like this. Is it ok? 
![image](https://github.com/purple-technology/phoenix-components/assets/29594184/45486db0-4c87-4622-bb8f-b558179cc0a5)

Reply:
File name should have size `sm`. Only the file size should be `xs`. Bigger filename, smaller size.

Apart from that - in the previews in Linear I was thinking the file name would be shown completely and would wrap on multiple lines, if necessary. But I don't have strong opinion about this. If you feel like it's better to truncate it, let's leave it like that.